lib/uiautomator2.js

```diff
@@ -305,10 +305,37 @@ class UiAutomator2Server {
   async cleanupAutomationLeftovers (strictCleanup = false) {
     this.log.debug(`Performing ${strictCleanup ? 'strict' : 'shallow'} cleanup of automation leftovers`);
 
+    const axiosTimeout = 500;
+
+    const waitStop = async () => {
+      // Wait for the process stop by sending a status request to the port.
+      // We observed the process stop could be delayed, then it caused the process stop
```

Comment:
We observed the process stop could be delayed, thus causing unexpected crashes in the middle of the session preparation process.